src/applications/lgy/coe/containers/introduction-content/COEStatuses/COEPending.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Telephone from '@department-of-veterans-affairs/component-library/Telephone';
+
+const COEPending = props => {
+  return (
+    <>
+      <va-alert status="warning">
+        <h3 slot="headline">
+          {props.pending
+            ? 'We need more information from you'
+            : 'We’re reviewing your application'}
+        </h3>
+        <p>You applied on:  June 30, 2020</p>
+        <div>
+          <p>
+            {props.pending
+              ? "You'll need to upload documents before we can make a decision on your COE application."
+              : "If you qualify for a Certificate of Eligibility, we'll notify you by email or mail to let you know how to get your COE."}
+          </p>
+          <a href="/housing-assistance/home-loans/apply-for-coe-form-26-1880/eligibility">
+            Go to your VA home loan COE page to see status details
+          </a>
+        </div>
+      </va-alert>
+      <div>
+        <h2>Should I apply again?</h2>
+        <p>
+          No. We’re reviewing your current application, and submitting a new
+          application won’t affect our decision or speed-up the process.
+        </p>
+        <p>
+          If more than 5 business days have passed since you submitted your
+          application and you haven’t heard back, please don’t apply again. Call
+          our toll-free number at <Telephone contact="8778273702" />.
+        </p>
+        <p>
+          We only recommend applying again if you’ve already worked with our VA
+          home loans case management team, and they’ve advised you to reapply.
+        </p>
+        <a href="#">Get instructions for downloading a VA.gov PDF</a>
```

Comment:
href here can be `/resources/how-to-download-and-open-a-vagov-pdf-form/`

Reply:
same as above